src/lensIndex.js

```diff
@@ -21,7 +21,5 @@ var nth = require('./nth');
  *     headLens.map(function(x) { return x + 1; }, [10, 20, 30, 40]); //=> [11, 20, 30, 40]
  */
 module.exports = function lensIndex(n) {
-  return lens(nth(n), function(x, xs) {
-    return _slice(xs, 0, n).concat([x], _slice(xs, n + 1));
-  });
+  return lens(nth(n), update(n));
 };
```

Comment:
This function is missing its `_curry1` wrapper. I thought we had a test in place to catch this. Apparently not.